ros/src/twist_controller/src/dbw_node_core.cpp

```diff
@@ -1,103 +1,98 @@
 #include "dbw_node_core.h"
 
-namespace DBWNODE_NS{
-
-DBWNode::DBWNode()
-: private_nh_("~"),
-sys_enable_(false)
-{
-    ROS_DEBUG("initForROS...");
-    initForROS();
-}
+namespace DBWNODE_NS {
 
-DBWNode::~DBWNode()
-{}
-
-void DBWNode::initForROS()
-{
-    private_nh_.param<double>("vehicle_mass", vehicle_mass_, 1736.35);
-    private_nh_.param<double>("fuel_capacity", fuel_capacity_, 13.5);
-    private_nh_.param<double>("brake_deadband", brake_deadband_, 0.1);
-    private_nh_.param<double>("decel_limit", decel_limit_, -5);
-    private_nh_.param<double>("accel_limit", accel_limit_, 1.0);
-    private_nh_.param<double>("wheel_radius", wheel_radius_, 0.2413);
-    private_nh_.param<double>("wheel_base", wheel_base_, 2.8498);
-    private_nh_.param<double>("steer_ratio", steer_ratio_, 14.8);
-    private_nh_.param<double>("max_lat_accel", max_lat_accel_, 3.0);
-    private_nh_.param<double>("max_steer_angle", max_steer_angle_, 8.0);
-
-    steer_pub_ = nh_.advertise<dbw_mkz_msgs::SteeringCmd>("/vehicle/steering_cmd", 2);
-    throttle_pub_ = nh_.advertise<dbw_mkz_msgs::ThrottleCmd>("/vehicle/throttle_cmd", 2);
-    brake_pub_ = nh_.advertise<dbw_mkz_msgs::BrakeCmd>("/vehicle/brake_cmd", 2);
-
-    sub_vel_ = nh_.subscribe("/twist_cmd", 2, &DBWNode::cbFromTwistCmd, this);
-    sub_enable_ = nh_.subscribe("/vehicle/dbw_enabled", 2, &DBWNode::cbFromRecvEnable, this);
-    sub_cur_vel_ = nh_.subscribe("/current_velocity", 2, &DBWNode::cbFromCurrentVelocity, this);
+    DBWNode::DBWNode()
+    : private_nh_("~"),
+    sys_enable_(false) {
+        ROS_DEBUG("initForROS...");

```

Comment:
I cannot see this text in the command line, is this code being executed?

Reply:
sure. it is debug mode info.